tests/rest/ssh_dashboard.json

```diff
@@ -1,285 +1,16 @@
 {
-  "title": "Your Basic Dashboard",
-  "services": {
-    "query": {
-      "list": {
-        "0": {
-          "query": "*",
-          "alias": "",
-          "color": "#7EB26D",
-          "id": 0,
-          "pin": false,
-          "type": "lucene",
-          "enable": true
-        }
-      },
-      "ids": [
-        0
-      ]
-    },
-    "filter": {
-      "list": {},
-      "ids": []
+  "dashboard": {
+    "title": "Your Basic Dashboard",
+    "hits": 0,
+    "description": "",
+    "panelsJSON": "[]",
+    "optionsJSON": "{\"darkTheme\":false,\"useMargins\":true,\"hidePanelTitles\":false}",
+    "version": 1,
```

Comment:
version should be removed, this has caused problems in the past as kibana likes to control version itself, it will automatically assign the version on it's own.

Reply:
Good catch, I'll remove it!